fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/boot/db/DataSourceProperties.java

```diff
@@ -31,8 +31,7 @@
  * -Dfineract.datasource.port=3307.
  */
 // NOT a @Component - we do not want this to picked up by component scan, only explicitly declared in DataSourceConfiguration (if that's active)
-public class DataSourceProperties extends PoolProperties {
-
+public class DataSourceProperties extends HikariConfig {
```

Comment:
If you manage to delete `DataSourceConfiguration.java` (above), then I suspect we can delete this entire class as well.. Or is there anything else left that uses this?